docs/1.4/extensions/front-matter.md

```diff
@@ -0,0 +1,176 @@
+---
```

Comment:
This shouldn't be in the 1.4 folder...

It also contains the configuration stuff that isn't implemented in this PR.  Need to remove this file.